src/tls/cockpit-certificate-helper.in

```diff
@@ -81,6 +81,9 @@ cmd_selfsign() {
     test ! -e "${CA_FILE}" || install_cert "${CA_FILE}"
     install_cert "${CERTFILE}"
     install_key "${KEYFILE}"
+
+    # Force writing of new files on disk.
+    sync
```

Comment:
Please move that into install_{cert,key} and call it with the file name, so that it only syncs that specific file, not everything.

Reply:
I was afraid of older versions of `sync(1)` that didn't take arguments: [old](https://linux.die.net/man/1/sync) vs [new](https://man7.org/linux/man-pages/man1/sync.1.html)

I've just done the test with an old debian 7 (using podman) and busybox (on my embedded device): it works, with an error trace: `sync: ignoring all arguments`,  and the return value is ok.

I will update as requested.

Reply:
Thanks! The oldest thing that we support is Ubuntu 22.04 (and RHEL 8 up to some degree). That supports a file argument.